recipes/qtsass/meta.yaml

```
@@ -0,0 +1,44 @@
+{%- set name = "qtsass" -%}
+{%- set version = "0.1.1" -%}
+{%- set hash = "4b1491e9810c7216365647e416fd7b49aeadc1ac25add8c6131deddf7549a072" -%}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ hash }}
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+  noarch: python
```

Sorry I missed this first time I looked, you should define an [`entry_point`](https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html?highlight=entry_points#python-entry-points) as there are [`console_scripts` in the `setup.py`](https://github.com/spyder-ide/qtsass/blob/master/setup.py#L56-L59):
```suggestion
  noarch: python
  entry_points:
    - qtsass = qtsass.cli:main
```

---

Thanks!